.changeset/switch-to-n-api.md

```diff
@@ -0,0 +1,5 @@
+---
+"ctrlc-windows": patch
```

Comment:
I think we can maybe have a discussion about this. It is technically just fixing a bug, but this is a big enough change that I could be convinced that we should do a `minor` or a `major` bump.